src/core/server/saved_objects/types.ts

```diff
@@ -72,6 +72,13 @@ export interface SavedObjectsPitParams {
   keepAlive?: string;
 }
 
+export interface SavedObjectsFindSearchOption {
+  search?: string;
+  searchFields?: string[];
+  rootSearchFields?: string[];
+  defaultSearchOperator?: 'AND' | 'OR';
```

Comment:
Can we get some insight on what guided you regarding what you did put in this new struct? Why not adding `hasReference` and `hasReferenceOperator` for example?